locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
+
+The CommComm has authority over community outreach including:
+ - Community Evangelism
+ - Education Initiatives
+ - Cultural Direction of Node.js Foundation
+ - Community Organization Outreach
+ - Project Moderation/Mediation
+ - Public Outreach and [Publications](https://medium.com/the-node-js-collection)
+
+There are four levels of engagement with the Community Committee:
+
+ 1. A **Contributor** is any individual creating or commenting on an issue or pull request.
+ 2. A **Collaborator** is a contributor who has been given write access to the repository
+ 3. An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+ 4. A **Member** is a collaborator with voting rights who has met the requirements of participation and voted in by the CommComm voting process.
+
+For the current list of Community Committee members, see the project's [README.md](https://github.com/nodejs/community-committee).
+
+## Collaborators
+
+It is the mission of CommComm to further build out the Node.js Community. If you're reading this, you're already a part of that community – and as a part of the Node.js Community, we'd love to have your help!
+
+The [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub repository is a great place to start. Check out the [issues labeled "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue) to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue.
+
+If you're interested in participating in the Community Committee as a member, you should create an issue asking to be an Observer in our next Community Committee meeting. You can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142).
+
+Before diving in though, please be sure to read the [Collaborator Guide](https://github.com/nodejs/community-committee/blob/master/COLLABORATOR_GUIDE.md).
+
+## Membership
+
+To become a member of the Community Committee, start participating! You must:
```

Comment:
Probably a better section intro to be had here.

Reply:
>If you're interested in participating in the Community Committee as a member, you can start by:

Suggested change - totally open to feedback/iterations.